modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= Cluster role bindings for unauthenticated groups
+
+There are use cases where it may be necessary to add `system:unauthenticated` to a cluster role.
+
+{product-title} {product-version} does not allow unauthenticated groups default access to cluster roles for security reasons.
```

Comment:
I wanted to break up _unauthenticated groups default access_ as it seemed a touch word-salad. Also, what about putting _For security reasons_ first to make it more important? 
```suggestion
For security reasons {product-title} {product-version} does not allow unauthenticated groups to have default access to cluster roles.
```